src/test/java/com/marklogic/kafka/connect/source/ReadRowsViaOpticDslTest.java

```diff
@@ -46,7 +46,8 @@ void noRowsReturned() throws InterruptedException {
             "expects no data to be returned.");
     }
 
-    @Test
+//    @Test
+// Not sure if we can test this now
     void testDslThatDoesNotStartWithFromView() {
```

Comment:
I'm thinking that in the case of a bad config - i.e. a bad Optic plan - we should still fail fast on the `start()` method. So I'm thinking we should call `getNewRowBatcher` in `start()` to ensure that it can least be created, and then call `stopJob` on it right away. Though it looks like it should still check for the "known server bug" (when no rows exist - that shouldn't throw an error). 

Also, now that I say that above - for consistency, just do `newRowBatcher` for that method name, which is consistent at least with stuff I've been doing in the Java Client and some other projects. I like to reserve `get` for pure getters that just retrieve a value.

Reply:
RowBatcher is not instantiated until poll() is called. So, we don't know if the Optic plan is bad until that point.
The call to start() doesn't even fail when I set the MarkLogic port incorrectly. Building the DatabaseClient must not actually talk to the database.

Reply:
getNewRowBatcher changed to newRowBatcher

Reply:
We agreed to omit this for now, as ML may simply not be available during start(), which could be acceptable

Reply:
test changed to verify that when the DSL is invalid, poll() returns null